### PR TITLE
Balance changes

### PR DESCRIPTION
**Pull request by taelon875**
Balanced Benevolence Healing. Buffed Premonition.

### CODE REVIEW

File: SWLOR.Game.Server/Feature/StatusEffectDefinition/PremonitionStatusEffect.cs [hunk 1]
```diff
@@ -45,11 +45,11 @@ private void Premonition1()
                 .CannotReplace(StatusEffectType.Premonition2)
                 .GrantAction((source, target, length, effectData) =>
                 {
-                    Impact(source, 5);
+                    Impact(source, 15);
```
Comment (zunath) [inline SWLOR.Game.Server/Feature/StatusEffectDefinition/PremonitionStatusEffect.cs:48]: You need to update the Perk definition to reflect this change. Also need updates to the Bible.

File: SWLOR.Game.Server/Feature/AbilityDefinition/Force/BenevolenceAbilityDefinition.cs [hunk 2]
```diff
@@ -43,7 +43,7 @@ private void Impact(uint activator, uint target, int baseAmount)
                 effect = TagEffect(effect, BeneRegen);
                 ApplyEffectToObject(DurationType.Temporary, effect, target, duration);
             }
-            var willHeal = baseAmount + targetBonus * 4 + Random.D10(targetBonus * 3);
+            var willHeal = baseAmount + (targetBonus * 4) + Random.D4(targetBonus);
```
Comment (zunath) [inline SWLOR.Game.Server/Feature/AbilityDefinition/Force/BenevolenceAbilityDefinition.cs:46]: Make sure the Perk definition is updated to reflect this change (if specifics are displayed). Update the Bible as appropriate too.

Comment (taelon875) [reply SWLOR.Game.Server/Feature/AbilityDefinition/Force/BenevolenceAbilityDefinition.cs:46]: None of this needs to be changed in definitions or the bible.

File: SWLOR.Game.Server/Feature/StatusEffectDefinition/PremonitionStatusEffect.cs (hunk 1 above)
Comment (taelon875) [reply SWLOR.Game.Server/Feature/StatusEffectDefinition/PremonitionStatusEffect.cs:48]: Fixed the perk definition, but not able to touch the bible. I know Skill issue.

**Review by zunath — APPROVED** [review]

**Pull request merged**